CommandHelper/LocalPackages/commands/chars/boostergold.yml

```diff
@@ -0,0 +1,33 @@
+costumes:
+- color: '&6'
+  name: §6B§1oos§6ter G§1ol§6d
+  sig: Dw4v3/Y7nRwa2eB46Rqx7A+6QsNhSDFp1nRD4RUD2MvfaQm7LpAQa34vhsuBPSOrp98WP3bUSsv+uftH6qaZiiOLOk/NmxxSOzgug5MnwEPSufpdhsthYSOXY+EBwUhN1uvRrTRZo/+wE3xkesKWtYYQkPGd8q5TBHoB9ey5jk/Z8F2iUR2E7qChqYl9rlnwcb0RPrZn/WhVc6XDW2vngiMEZdncXakjpkPgaGL48LLPDtWRSgSe5mAtNvqTocvFcdWvPd9cPlbS130w6dH/IU/WuX3rGcBFAJjDzQfrHofQgLYOrvIz61tMSovCvHCYEcThxX/FHnAfdwDGmBkAUVOq8IAqEO5K2Xly6d4vbA3ZLoTndHbJWfGcDq/8HXVct26oh1OXAQTSemexM6qRJAGrI0u+dTrrkYSDIPjCjIIeev7lpUzwY3m7a/hnxr8s3ZJ+NR9lr/L3D34qfiDIdNh5oObQmlYtzIwDHCSq5W1QES3B7APTdDMTuJQa5bpoCNuqeZKw+yiVyWY8psWjT3u3ztebMskq2WaGQx1C5hpsAGjrHS1qa6G8wrpOTGRrXZm/12eNv1SMr/tPf56k1Uh30IGVH5MpY7FBf4/L+QGTdgTrPwAEGlk4nySVtI0tLrIc0ZBlrPkNK7HFhQ4l+r4iMoAHCBfKTe7oSmBqlKI=
+  val: eyJ0aW1lc3RhbXAiOjE1MjA4NDMyODkxNzIsInByb2ZpbGVJZCI6ImQxZmJhYWI3YmRlMzQ0MDU4MjgyOWZjMTVkOTAyMDlkIiwicHJvZmlsZU5hbWUiOiJUaGVEYXJrQXVyYV8iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlL2FiOWRlZmRhOWMyOGFkOWY0OGE0OWQ5NzliMzVmM2VhYTJhNmUxODYzM2NjNGJhYjY1MzgyNjI3YzhhZjZhYyJ9fX0=
+  buffs:
+- /effect <plr> 5 9999 5 true
+class: boostergold
+haste: 3
+maxHearts: 30
+op: false
+permissions:
+  superherocity:
+  - powers.nofalldamage
+  - -essentials.protect.damage.fall
```

Comment:
powers.nofalldamage does nothing, you can remove it
-essentials.protect.damage.fall is only for people without /fly. if they /fly, they already automatically get no fall damage.

that means you should remove -essentials.protect.damage.fall because he already has /fly